clients/benchmarks/client.cpp

```diff
@@ -823,15 +823,18 @@ try
 
     arg.bias_source = string_to_hipblaslt_bias_source(bias_source);
 
-    auto scaleString2Enum = [](std::string& s) {
-        if(s == "s")
-            return Arguments::ScalingFormat::Scalar;
-        if(s == "v")
-            return Arguments::ScalingFormat::Vector;
-        return Arguments::ScalingFormat::None;
+    auto scaleInt2Enum = [](int s) {
+        if(s == 0)
+            return hipblaslt_scaling_format::none;
+        if(s == 1)
+            return hipblaslt_scaling_format::Scalar;
+        if(s == 2)
+            return hipblaslt_scaling_format::Vector;
+
+        return hipblaslt_scaling_format::none;
```

Comment:
NBD, should we break here instead?

Reply:
rather than break, thought setting it to default: none would be the right choice.